packages/react-dom/src/__tests__/ReactDOMComponent-test.js

```diff
@@ -2283,7 +2286,9 @@ describe('ReactDOMComponent', () => {
 
       expectDev(console.error.calls.count()).toBe(1);
       expectDev(console.error.calls.argsFor(0)[0]).toContain(
-        'Warning: Received `false` for non-boolean attribute `whatever`.',
+        'If you mean to conditionally pass an attribute, use a ternary ' +
+          'expression: `false`={condition ? value : undefined} instead of ' +
+          '{condition && value}.',
```

Comment:
What does `false={condition? value : undefined}` mean? Why does it says `false`?

Please fix this to show the attribute name.